frontend/src/lib/components/Table.tsx

```diff
@@ -19,6 +20,7 @@ export function createdByColumn(items: Record<string, any>[]): Record<string, an
     const { user } = useValues(userLogic)
     return {
         title: 'Created by',
+        width: 150,
```

Comment:
I'm also slightly vary about imposing this. You might have a long (e.g. indian, russian, whatever) name that just breaks with "..." on the last letter. 

I'd be a much bigger fan of something like: "max-width: 300px, width: auto", though that might be annoyingly complicated to implement?

Reply:
Got it let's do that! It doesn't look too complicated actually